base/src/com/google/idea/blaze/base/dependencies/BlazeQueryDirectoryToTargetProvider.java

```diff
@@ -57,10 +57,10 @@ public List<TargetInfo> doExpandDirectoryTargets(
       ImportRoots directories,
       WorkspacePathResolver pathResolver,
       BlazeContext context) {
-    return runQuery(project, getQueryString(directories, shouldManualTargetSync), context);
+    return runQuery(project, getQueryString(directories, shouldManualTargetSync, pathResolver), context);
   }
 
-  protected static String getQueryString(ImportRoots directories, boolean allowManualTargetsSync) {
+  protected static String getQueryString(ImportRoots directories, boolean allowManualTargetsSync, WorkspacePathResolver pathResolver) {
```

Comment:
Can you make it public and annotate it with @VisibleForTesting instead of changing the package in the test target?

Reply:
done